doc/mgr/progress.rst

```diff
@@ -0,0 +1,30 @@
+Progress Module
+===============
+
+The progress module is primarily used for informing users about the recovery progress
+of PGs (Placement Groups) that are affected by events such as OSDs being marked in/out,
+pg_autoscaler trying to match the target PG number, balancer trying to perform 
+upmap/crush-compat optimization and etc. 
+
+Ongoing events and its progress can be seen in ``ceph -s`` and a more detailed version
+of this is outputted to the file mgr.x.log.
+
+Commands
+--------
+::
+
+        ceph progress
+
+Shows the summary of all the ongoing/completed events and its duration
+in chronological order.
```

Comment:
What is the difference to `ceph status`?

Reply:
@sebastian-philipp 
> What is the difference to `ceph status`?
ceph -s show events that has been completed, it only shows the events that are in progress